packaging/fedora/tracee.spec

```diff
@@ -75,21 +75,21 @@ ln -s %{_libdir}/tracee/tracee-rules $RPM_BUILD_ROOT/%{_bindir}/tracee-rules
 install -m 0755 ./dist/tracee $RPM_BUILD_ROOT/%{_libdir}/tracee
 ln -s %{_libdir}/tracee/tracee $RPM_BUILD_ROOT/%{_bindir}/tracee
 
-# rules
-mkdir -m 0755 -p $RPM_BUILD_ROOT/%{_libdir}/tracee/rules/
-install -m 0644 ./dist/rules/* $RPM_BUILD_ROOT/%{_libdir}/tracee/rules/
+# signatures
+mkdir -m 0755 -p $RPM_BUILD_ROOT/%{_libdir}/tracee/signatures/
+install -m 0644 ./dist/signatures/* $RPM_BUILD_ROOT/%{_libdir}/tracee/signatures/
 
 %clean
 
 %files
 %{_bindir}/tracee
 %{_libdir}/tracee/tracee
-%{_libdir}/tracee/rules*
+%{_libdir}/tracee/signatures*
 
 %files -n tracee-ebpf
 %{_bindir}/tracee-ebpf
 
 %files -n tracee-rules
```

Comment:
this should be named tracee-signatures (it will generate a tracee-signatures) package.. and all "tracee-rules" lines should be replaced by "tracee-signatures" as well. You can test the package generation by executing `make -f builder/Makefile.packaging fedora-bin-37` to make sure it works and generates the correct packages.

Reply:
Ok I see now the point you want to keep tracee-rules (since it will get deprecated). Lets check if packages are good..

Reply:
nm, looks ok, got the idea.

Reply:
👍